freud/order.pyx

```diff
@@ -543,11 +544,12 @@ cdef class Steinhardt(_PairCompute):
         to observe a value of 0 for the per-particle order parameter even with
         a finite number of neighbors. If you would like to ignore this
         distinction, you can mask the output order parameter values using
-        NumPy: :code:`numpy.nan_to_num(particle_order)`.
+        NumPy: :code:`numpy.nan_to_num(particle_order[0])`.
```

Comment:
I think the `[0]` is not helpful here -- it's assuming you have an array named `particle_order`.
```suggestion
        NumPy: :code:`numpy.nan_to_num(particle_order)`.
```